flytekit/extras/tensorflow/tensor/tensor_transformer.py

```diff
@@ -0,0 +1,82 @@
+import pathlib
+from typing import Type, TypeVar
+
+import tensorflow as tf
+
+from flytekit.core.context_manager import FlyteContext
+from flytekit.core.type_engine import TypeEngine, TypeTransformer, TypeTransformerFailedError
+from flytekit.models.core import types as _core_types
+from flytekit.models.literals import Blob, BlobMetadata, Literal, Scalar
+from flytekit.models.types import LiteralType
+
+T = TypeVar("T")
+
+
+class TensorflowTensorTransformer(TypeTransformer[tf.Tensor]):
+    """
+    TypeTransformer that supports tf.tensor as a native type
+    """
+
+    TENSORFLOW_FORMAT = "TensorflowTensor"
+
+    def __init__(self):
+        super().__init__(name="Tensorflow Tensor", t=tf.Tensor)
+
+    def get_literal_type(self, t: Type[T]) -> LiteralType:
```

Comment:
Can we have the actual type here rather than `T`? See https://github.com/flyteorg/flytekit/blob/b1ff43e3800500d8244c33f21e7635752fd8ee77/plugins/flytekit-onnx-pytorch/flytekitplugins/onnxpytorch/schema.py#L85-L140 for reference.